GHSA-ch8v-r6jh-4vvr: FILTER_SANITIZE_ENCODED left 0xFF to stack garbage; php-src 7ac9700d0d46

diff --git a/ext/filter/sanitizing_filters.c b/ext/filter/sanitizing_filters.c
index a8eb9539a..a60284dcb 100644
--- a/ext/filter/sanitizing_filters.c
+++ b/ext/filter/sanitizing_filters.c
@@ -71,7 +71,7 @@ static void php_filter_encode_url(zval *value, const unsigned char* chars, const
 	unsigned char *s = (unsigned char *)chars;
 	unsigned char *e = s + char_len;
 
-	memset(tmp, 1, sizeof(tmp)-1);
+	memset(tmp, 1, sizeof(tmp));
 
 	while (s < e) {
 		tmp[*s++] = 0;
